apa102/softspi.go

```diff
@@ -1,24 +1,30 @@
 package apa102
 
-import "machine"
+import (
+	"tinygo.org/x/drivers"
+	"tinygo.org/x/drivers/internal/legacy"
+)
 
 // bbSPI is a dumb bit-bang implementation of SPI protocol that is hardcoded
 // to mode 0 and ignores trying to receive data. Just enough for the APA102.
 // Note: making this unexported for now because it is probable not suitable
 // most purposes other than the APA102 package. It might be desirable to make
 // this more generic and include it in the TinyGo "machine" package instead.
 type bbSPI struct {
-	SCK   machine.Pin
-	SDO   machine.Pin
-	Delay uint32
+	SCK    drivers.PinOutput
+	SDO    drivers.PinOutput
+	Delay  uint32
+	config func()
```

Comment:
```suggestion
	configure func()
```
reasoning:
slightly the same as for remark to "SCK" - config() is the getter function for the "config"-attribute or object, but we like to configure something here, so we need to use the verb for the function name

Reply:
Applied change, now is configurePins, which is even more descriptive